_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1386,9 +1389,29 @@ cloudtrail_s3_bucket_name = "<CLOUDTRAIL_BUCKET_NAME>"
 config_s3_bucket_name     = "<CONFIG_BUCKET_NAME>"
 ----
 
-Take note of all of this data, as you'll need it again shortly!
+If you followed the http://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#prepare-your-infrastructure-live-repository[steps for preparing your `infrastructure-live` repo],
+now you can update the `accounts.json` file with the account IDs from the Terraform output! If you are instead making use of `account.hcl` files (located in each account folder (e.g., `infrastructure-live/dev`, `infrastructure-live/shared`, etc.), update them too with the
+appropriate account ID shown in the Terraform output, so they look like:
+[source,hcl]
+----
+locals {
+  account_name = "<REPLACE_WITH_NAME_OF_ACCOUNT>"
+  account_id = "<REPLACE_WITH_ID_OF_ACCOUNT>"
+}
+
+----
 
-One other useful output will be the encrypted passwords for any IAM users you created:
+[source,json]
+----
+{
+  "account_name": "<REPLACE_WITH_ID_OF_ACCOUNT>"
+}
+----
+
+Also update the entries for `cloudtrail_kms_key_arn`, `cloudtrail_s3_bucket_name`, and `config_s3_bucket_name` into your
```

Comment:
The bucket names need to be defined from the beginning, but not `cloudtrail_kms_key_arn`,  so perhaps they live in the common file since the root apply

Reply:
oh, I see - do you mean that `cloudtrail_kms_key_arn` stays here, but we update the `common.hcl` to have `cloudtrail_s3_bucket_name` and `config_s3_bucket_name`?

Reply:
Yeah, the `account-baseline-root` will need these values as input, while the arn is an output